patches/chrome-browser-performance_manager-chrome_browser_main_extra_parts_performance_manager.cc.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc b/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc
+index 11548a8313dc775cb9bea3bc041d896e42ce305a..bbdced4ba79bd5722b74b0639e1c2bcfa7b6f250 100644
+--- a/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc
++++ b/chrome/browser/performance_manager/chrome_browser_main_extra_parts_performance_manager.cc
+@@ -310,11 +310,13 @@ void ChromeBrowserMainExtraPartsPerformanceManager::PreMainMessageLoopRun() {
+   performance_manager::user_tuning::UserPerformanceTuningManager::GetInstance()
+       ->Start();
+ 
++#if 0  // Disabled in Brave
+   // This object is created by the metrics service before threads, but it
+   // needs the UserPerformanceTuningManager to exist. At this point it's
+   // instantiated, but still needs to be initialized.
+   performance_manager::MetricsProviderDesktop::GetInstance()->Initialize();
```

Comment:
can we chromium_src MetricsProviderDesktop to make `Initialize()` a noop, instead of this patch?

Reply:
The issue is `GetInstance() == nullptr`, not with `Initialize()`.

We can try to use `chromium_src` for this file to replace `MetricsProviderDesktop`  to some`Fake MetricsProviderDesktop`

Reply:
Removed the patch in favor of chromium_src override.